website/content/en/pre-docs/tasks/provisioning-task.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Provisioning"
+linkTitle: "Provisioning"
+weight: 5
+---
+
+When you first installed Karpenter, you set up a default [Provisioner](../getting-started/#provisioner).
+The Provisioner sets constraints on the nodes that can be created by Karpenter and the pods that can run on those nodes.
+The Provisioner can be set to do things like:
+
+* Define taints to limit the pods that can run on nodes Karpenter creates
+* Limit nodes creation to certain zones, instance types, and computer architectures
+* Set defaults for node expiration
+
+You can change your provisioner or add other provisioners to Karpenter.
+Here are things you should know about Provisioners:
+
+* Karpenter won't do anything if there is not at least one Provisioner configured.
+* Each Provisioner that is configured is looped through by Karpenter.
+* If Karpenter encounters a taint in the Provisioner that is not tolerated by a Pod, Karpenter won't use that Provisioner to provision the pod.
+* It is best to not create Provisioners that are mutually exclusive. So no Pod should match multiple Provisioners. If multiple Provisioners are matched, Karpenter will randomly choose which to use.
+
+If you want to modify or add provisioners to Karpenter, do the following:
+
+1. Review the following Provisioner documents:
+  * [Provisioner](../getting-started/#provisioner) in the Getting Started guide for a sample default Provisioner.
+  * [Provisioner API](../provisioner-crd) for a description of Provisioner custom resource.
+  * [Provisioning Configuration](../AWS/constraints) for cloud-specific settings.
+
+1. Apply the new or modified Provisioner to the cluster.
+
+The following examples illustrate different aspects of Provisioners.
+Refer to [Running pods](../tasks/running-pods) to see how the same features are used in Pod specs to determine where pods run.
+
+## Example: Requirements
+
+This provisioner limits node selection to those in US West availability zones.
+It allows both spot and on-demand capacity types, but prefers spot.
```

Comment:
This might be a bug, but we currently prefer OD unless spot is explicitly requested: https://github.com/aws/karpenter/blob/ce2b1a37179b6877536b4b81c27c5784a70766a8/pkg/cloudprovider/aws/instance.go#L114

Reply:
I'll drop the ", but prefers spot".